clustering/ee/cache/src/main/java/org/wildfly/clustering/ee/cache/scheduler/FastConcurrentDirectDeque.java

```diff
@@ -1157,7 +1149,7 @@ public boolean addAll(Collection<? extends E> c) {
         // Copy c into a private chain of Nodes
         Node<E> beginningOfTheEnd = null, last = null;
         for (E e : c) {
-            checkNotNull(e);
+            checkNotNullParamWithNullPointerException("e", e);
```

Comment:
"e" is not a parameter of this method. Is there an Assert.checkNotNullArrayParam(...) equivalent for collections?